src/olympia/users/forms.py

```diff
@@ -198,6 +195,10 @@ class Meta:
         model = UserProfile
         exclude = USER_EDIT_EXCLUDE_FIELDS
 
+    def clean_email(self):
+        # TODO(django 1.10): Change the field to disabled=True and remove this.
```

Comment:
Not sure if you mean HTML disabled but if you do, one thing to note is that would prevent copy/paste on the field value. You can copy/paste a readonly field though. Not sure if that matters but disabled usually seems too brutal for most cases.